hydra-node/src/Hydra/Chain/Direct/Handlers.hs

```diff
@@ -273,7 +273,8 @@ fromPostChainTx timeHandle wallet ctx cst@ChainStateAt{chainState} tx = do
       pure $ collect ctx st
     (CloseTx{confirmedSnapshot}, Open st) -> do
       shifted <- throwLeft $ adjustPointInTime closeGraceTime pointInTime
-      pure (close ctx st confirmedSnapshot shifted)
+      -- REVIEW: what do we use for lower bound slot here?
```

Comment:
The current slot (maybe name it like this) should be fine.

Reply:
But maybe.. it's not. It may be that the ledger is still not at the `currentPointInTime` and the node might reject this slot.

Reply:
Option 1: Change `queryTimeHandle` to use the latest slot known to the node as `currentPointInTime`. BUT we need to check all usages of time handle whether this change in semantics is fine!

Option 2: Use the upper bound (`shifted` here) and adjust get a slot number within the allowed window (= contestation period), i.e. something like `startSlotNo = upperSlotNo - contestionPeriodInSlots`. NOTE we need access to the right contestation period for this option.

Reply:
Note to self: I didn't explore but want to the Option no 1 here